README.md

```diff
@@ -30,13 +30,13 @@ docker run -d -p 3306:3306 -p 9000:9000 --name=memsql memsql/quickstart
 On Mac
 
 ```
-open "http://$(boot2docker ip):9000"
+open "http://$(docker inspect --format '{{ .NetworkSettings.IPAddress }}' memsql):9000"
```

Comment:
you may want to make a note that this depends on the user setting up routes from their mac/pc to the docker VM